server/meshmodel/kubernetes/v1.25.2/v1.0.0/relationships/hierarchical-inventory-configMap-to-deployment.json

```diff
@@ -42,7 +42,13 @@
               "mutatorRef": [
                 [
                   "displayName"
+                ],
+                [
+                  "configuration",
+                  "metadata",
```

Comment:
Ensure this particular `mutatorRef` is present for all relationships involving namespaced resources.